src/reflect/scala/reflect/internal/transform/Erasure.scala

```diff
@@ -388,6 +446,191 @@ trait Erasure {
     }
   }
 
+  /** Scala 3 implementation of erasure for intersection types.
+   *  @param components the erased component types of the intersection.
+   */
+  def erasedGlb(components: List[Type]): Type = {
```

Comment:
move this method inside `Scala3ErasureMap`, since it's only used there?

Reply:
done